### ISSUE TITLE
Backups failing when using start-fast option

### ISSUE THREAD

**Issue by golpa** [pgbackrest/pgbackrest#323]
Hi,
  We've been using pgbackrest for almost 2 weeks now and everything was working well until a couple of days ago when a backup failed with:
```
-------------------PROCESS START-------------------
2017-02-10 00:15:01.285 P00   INFO: backup command begin 1.13: --backup-standby --compress-level=6 --compress-level-network=6 --db-host=primary --db-path=/pgdb/pgdata --db-socket-path=/tmp --db-user=postgres --db2-host=secondary --db2-path=/pgdb/pgdata --db2-socket-path=/tmp --db2-user=postgres --process-max=12 --repo-path=/var/lib/pgbackrest --retention-full=3 --stanza=laads --start-fast --type=incr
2017-02-10 00:15:02.640 P00   INFO: last backup label = 20170205-032609F_20170209-020720I, version = 1.13
2017-02-10 00:15:06.546 P00   INFO: execute exclusive pg_start_backup() with label "pgBackRest backup started at 2017-02-10 00:15:01": backup begins after the requested immediate checkpoint completes
2017-02-10 00:15:08.491 P00   INFO: backup start archive = 000000010000626A000000A8, lsn = 626A/A80288A8
2017-02-10 00:15:08.829 P00   INFO: wait for replay on the standby to reach 626A/A80288A8
2017-02-10 00:15:12.740 P00   INFO: replay on the standby reached 626A/A84AB560
2017-02-10 00:15:13.647 P00  ERROR: [117]: raised on primary host: /pgdb/pgdata/base/16422/t214_3552907721 could not be read: No such file or directory
2017-02-10 00:15:13.668 P00  ERROR: [117]: raised on primary host: /pgdb/pgdata/base/16422/t214_3552907721 could not be read: No such file or directory
2017-02-10 00:15:13.668 P00   INFO: backup command end: aborted with exception [117]
```

ever since then all backups fail with error message (full or incremental backups):
```
ERROR: [132]: raised on primary host: ERROR:  a backup is already in progress
```

If I try to perform a full or incremental backup using `--stop-auto=y` the backup fails with similar error message as the original failure (just different files each time):
```
ERROR: [117]: raised on primary host: /pgdb/pgdata/base/16422/<file> could not be read: No such file or directory
```

It appears the fix was to remove the `start-fast=y` from `pgbackrest.conf` file.

**Comment by dwsteele** [comment]
It appears this is being caused by a regression in 1.13 and is not related to `--stop-fast` or `--stop-auto`.  I think you just got lucky the last time.  Your backup should be good, the problem is that pgBackRest is not ignoring files that go missing while the backup manifest is being built.  Since this only takes a few seconds it won't always cause an issue, even on a busy system.

I'll prepare a fix for this, but meanwhile you can revert to 1.12 if you want to keep backups running.

**Comment by MichaelDBA** [comment]
Does that mean the best course of action is not to upgrade to 1.13, but wait for 1.14?

**Comment by dwsteele** [comment]
@golpa How would you like your name to appear in the release notes?

**Comment by dwsteele** [comment]
@MichaelDBA Actually, you'll want to wait for 1.15.  1.14 was released this morning and has the same issue.

**Comment by golpa** [comment]
@dwsteele That's fine with me. Thanks.

**Comment by dwsteele** [comment]
@golpa Just to clarify, "golpa" is fine for the release notes?

**Comment by golpa** [comment]
Yup or you can use my full name, Navid Golpayegani. Whichever format you prefer.

**Comment by dwsteele** [comment]
Fixed at 2d7d9626bb60640a77ac71559f2a714a1aac8b94.

I have also added a bunch of new tests to hopefully prevent this regression from happening again.
